pkg/xray/resource_xray_license_policy_test.go

```diff
@@ -0,0 +1,361 @@
+package xray
+
+import (
+	"fmt"
+	"net/http"
+	"regexp"
+	"strconv"
+	"testing"
+
+	"github.com/go-resty/resty/v2"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+// License policy criteria are different from the security policy criteria
+// Test will try to post a new license policy with incorrect body of security policy
+// with specified cvss_range. The function expandLicenseCriteria will ignore all the
+// fields except of "allow_unknown", "banned_licenses" and "allowed_licenses" if the Policy type is "license"
+func TestAccLicensePolicy_badLicenseCriteria(t *testing.T) {
+	policyName := "terraform-license-policy-1"
+	policyDesc := "policy created by xray acceptance tests"
+	ruleName := "test-license-rule-1"
+	rangeTo := 5
+	resourceName := "policy-" + strconv.Itoa(randomInt())
```

Comment:
a more terse version would be:
`resourceName := fmt.Sprintf("policy-%d",randomInt())`

Reply:
_, fqrn, resourceName := mkNames("policy-", "xray_license_policy")
even more terse